CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ For information on changes in released versions of Teku, see the [releases page]
 
 ### Additions and Improvements
  - Log a warning instead of a verbose error if node is syncing while performing sync committee duties
+ - Distributions created from the same git commit will have exactly the same contents, *i.e.*, they are reproducible
```

Comment:
We possibly should list this as being identical when the distribution is built using the same docker image as well as the same git commit.

Reply:
Yes, I agree. Will make that change now.